lib/src/test/java/com/thenewboston/utils/Mocks.kt

```diff
@@ -110,4 +113,19 @@ object Mocks {
     )
 
     fun internalServerError() = BankAPIError(500, "Internal Server Error")
+
+    fun bankTrustRequest() = BankTrustRequest(
+        Message(10.0),
+        "35f4c988f425809ca7f5d0b319cdf8f7d7aba1b064fd0efc85d61fa0f4d05145",
+        "93952df29ae3885fd9c9f88721314236bdb53ca5632b2959dcf5cf3c38cb8b96ca57ff84c5337eb164f803237f901abcb0c41a9f71e14aa2fb3159c7ad7a7509"
+    )
+
+    fun bankTrustResponse() = BankTrustResponse(
+        "dfddf07ec15cbf363ecb52eedd7133b70b3ec896b488460bcecaba63e8e36be5",
+        "127.0.0.1",
+        80,
+        "http",
+        1.0,
+        10.0
```

Comment:
I was thinking about doing the same in #65 but then what are we still testing? Our implementation could be sending some random trust every time but we're asserting that it should be 10 afterwards and mocking the 10 in here. So I've added a bit of logic into the mock in my branch. We can discuss that once I have my PR ready :)

Reply:
Hmm.. I get you. So you're saying there should be a min and maximum level of trust a user can send? If this is not what you meant, I'll wait for the PR :)

Reply:
I meant something else, we can discuss it on my PR ;) Yours is another interesting point...but I'd expect the API to just return some error, maybe 400 Bad Request if only certain trust levels should be allowed so not sure if we want to restrict in on our end.

Reply:
Trusts should be a `string` in which `trust` as a `float` satisfies `trust >= 0.0` and `trust <= 100.0`.  I dunno if the client should throw an error on the trust being out of bounds, though.